package.json

```diff
@@ -14,5 +14,11 @@
   "license": "MIT",
   "devDependencies": {
     "concurrently": "^8.0.1"
+  },
+  "dependencies": {
```

Comment:
it is root directory, revert packages changes, also regenerate package-lock.json